source/app/metrics/setup.mjs

```diff
@@ -40,18 +40,28 @@ export default async function({log = true, sandbox = false, community = {}} = {}
   }
 
   //Load settings
-  logger("metrics/setup > load settings.json")
-  if (fs.existsSync(__settings)) {
-    if (sandbox)
-      logger("metrics/setup > load settings.json > skipped because in sandbox is enabled")
-    else {
-      conf.settings = JSON.parse(`${await fs.promises.readFile(__settings)}`)
-      logger("metrics/setup > load settings.json > success")
+  {
```

Comment:
why the extra level?

Reply:
it's just to scope the `fd`, since it needs to be outside of the `try {}` so it can be referenced by `finally {}`, but the rest of the function shouldn't reference it

actually the solution would be to refactor this function in smaller chunks, but I don't have the courage to do so